.githooks/commit-msg

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/sh
+
+declare -A components
+components=( \
+	["cnf-tests"]="cnf-tests","go.mod","go.sum" \
```

Comment:
go.mod / go.sum may impact the policy generator too

Reply:
so will keep them in exceptions list then

Reply:
actually maybe better to add it in ztp too

Reply:
and I'm contemplating the vendor directory
if it need its own component or be part of these 2 too.

Reply:
hmm actually, the other way around
I'll keep the vendor component
but allow `vendor:` commits to touch cnf-tests and ztp too (so when updating vendoring, must do that with proper component but can also update relevant go files